.ci/integration/vars/Ubuntu.yml

```diff
@@ -6,6 +6,7 @@ test_dependencies:
   - man
   - ruby-dev
   - rubygems
+  - rsync
```

Comment:
Which is preferable: adding rsync in this file, or in 
.Dockerfiles/ubuntu/latest/Dockerfile <https://github.com/sobolevn/git-secret/pull/181/commits/2f584f26c5bfcef06e01b009aab448123cbf9803>